curator-x-discovery-server/pom.xml

```diff
@@ -127,4 +127,41 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>jdk-9-plus</id>
```

Comment:
I'm -1 on this change. I'd rather see a comprehensive Java 11 PR (we have a Jira for this already). Java 9 is not an LTS release.

Reply:
This profile is activated whenever the test runtime is Java 9+ (including Java 11). I could change this to Java 11+, so that Java 9 and 10 are not supported at all, but I don't see much sense in it. In fact, it will break the workflow for people who want to run some tests locally and for whatever reason they happen to have Java 9 or 10, but not Java 11 installed (that was my case).